UnityProject/Assets/Scripts/FloorDecals/FloorDecal.cs

```diff
@@ -50,7 +50,7 @@ public override void OnStartClient()
 
 	private void SyncChosenSprite(int _chosenSprite)
 	{
-		chosenSprite = chosenSprite;
+		chosenSprite = chosenSprite; // TODO Fix this compiler warning or remove this line.
```

Comment:
what the! I think this was meant to be chosenSprite = _chosenSprite

Reply:
Thought the same thing, but I'm not familiar with how floor decals are supposed to work.

Would you like me to change or remove the line?

Reply:
yep if you can make the change I'll merge this